styles.css

```
@@ -18,50 +26,397 @@ body {
     font-family: 'Roboto', sans-serif;
 }
 
-/* ------------------------------- 3. Layout --------------------------------------- */
+/* ------------------------------- 4. Areas ---------------------------------------- */
 
-/* --------------- 3.1 Reusables ----------------------- */
+/* --------------- 4.1 Header ------------------------- */
 
 .outer-content-container {
-    width: 100%;
+    /*width: 100%;*/
+    height: 600px;
     display: flex;
     justify content: center;
 }
 
 .inner-content-container {
     display: flex;
-    flex basis: 1400px;
+    flex basis: 1200px;
     flex shrink: 1;
     flex grow: 0;
     flex direction: column;
     margin: 0 50px;
 }
 
-/* --------------- 3.2 Other --------------------------- */
-
 /* ------------------------------- 4. Areas ---------------------------------------- */
 
 /* --------------- 4.1 Header ------------------------- */
 
 .header-content_navigation {
-    border: 2px dashed
+    background: #4845E4;
     display: flex;
     flex-direction: row;
     justify-content: flex-end;
     widht: 100%;
 }
 
 .header-menu_list {
-    border: 2px dashed orange;
+    background: #4845E4;
+    color: white;
     flex-basis: 400px;
-    min-height: 100px;
-    flex-shrink: 1
-    flex-grow: 0
+    min-height: 25px;
+    flex-shrink: 1;
+    flex-grow: 0;
     list-style-type: none;
     display: flex;
     justify-content: space-evenly;
 }
 
-/* --------------- 4.2 Introduction ------------------- */
+.header-menu_item_1 a  {
+    color: white;
+    text-decoration: none;
+}
+
+.header-menu_item_2 a {
+    color: white;
+    text-decoration: none;
+}
+
+.header-menu_item_3 a {
+    color: white;
+    text-decoration: none;
+}
+
+.header-content__cta {
+    flex-basis: 100vw;
+    background: linear-gradient(0deg, rgba(87,34,205,1) 0%, rgba(72,69,228,1) 100%);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    color: white;
+
+    position: relative;
+    bottom: 0;
+}
+
+.header-content__cta h1 {
+    font-family: Merriweather, sans-serif;
+    color: #ffffff;
+    font-size: 3.4rem;
+    /*font-size: 60px;*/
+    font-weight: 500;
+    text-align: center;
+}
+
+.header-content__cta h2 {
+    text-align: center;
+    color: #ffffff;
+    /*font-size: 1.75rem;*/
+    font-size: 30px;
+    font-weight: 300;
+    line-height: 2em;
+}
+
+.button_cta {
+    font-size: 18px;
+    font-weight: 400;
+    text-transform: uppercase;
+    color: #ffffff;
+    background-color: #ed4385;
+    border: #ed4385;
+    border-radius: 50px 50px 50px 50px;
```

Dit kun je korter schrijven:
```suggestion
    border-radius: 50px;
```